src/main/java/uk/gov/hmcts/reform/civil/validation/UnavailableDateValidator.java

```diff
@@ -50,34 +52,76 @@ public boolean isValid(UnavailableDate value, ConstraintValidatorContext context
         return errors;
     }
 
+    public List<String> validateFastClaimHearing(HearingLRspec hearingLRspec) {
```

Comment:
Can we do anything about reducing the code duplication (aware of the type constraints) with respect to the next method validateSmallClaimsHearing.

Reply:
incorporated

Reply:
refactor not null or empty code

Reply:
change incorporated